### PR TITLE
Move maps from previewers to feature flag

### PR DESCRIPTION
**Pull request by orangejenny**
http://manage.dimagi.com/default.asp?171181

@nickpell / @czue

### CODE REVIEW

File: corehq/apps/reports/standard/maps.py [hunk 1]
```diff
@@ -479,7 +480,7 @@ class DemoMapReport(GenericMapReport):
 
     @classmethod
     def show_in_navigation(cls, domain=None, project=None, user=None):
-        return user and user.is_previewer()
+        return user and toggles.DEMO_REPORTS.enabled(user.username) or domain and toggles.DEMO_REPORTS.enabled(domain)
```
Comment (dimagimon) [inline corehq/apps/reports/standard/maps.py]: E501 line too long (118 > 115 characters)

File: corehq/apps/reports/standard/maps.py [hunk 2]
```diff
@@ -560,7 +561,7 @@ class DemoMapReport2(GenericMapReport):
...
```
Comment (dimagimon) [inline corehq/apps/reports/standard/maps.py]: E501 line too long (118 > 115 characters)

File: corehq/apps/reports/standard/maps.py [hunk 3]
```diff
@@ -637,7 +638,7 @@ class DemoMapCaseList(GenericCaseListMap):
...
```
Comment (dimagimon) [inline corehq/apps/reports/standard/maps.py]: E501 line too long (118 > 115 characters)

**Comment by czue** [comment]
good stuff

**Pull request merged**